src/compiler/transformers/ts.ts

```diff
@@ -2055,6 +2085,20 @@ namespace ts {
             }
         }
 
+        function getPropertyNameExpressionIfNeeded(name: PropertyName, shouldHoist: boolean, omitSimple: boolean): Expression {

```

Comment:
Add documentation to explain this function's purpose. Also `omitSimple` is not very clear, either give it a better name or document it.